code/code/disc/disc_offense.h

```diff
@@ -0,0 +1,24 @@
+#ifndef __DISC_OFFENSE_H
+#define __DISC_OFFENSE_H
+
+#include "discipline.h"
+#include "skills.h"
+
+class CDOffense : public CDiscipline
+{
+public:
+    CSkill skAdvancedOffense;
+
+    CDOffense();
+    CDOffense(const CDOffense &a);
+    CDOffense & operator=(const CDOffense &a);
+    virtual ~CDOffense();
+    virtual CDOffense * cloneMe() { return new CDOffense(*this); }
```

Comment:
Whoa, what's the `cloneMe` for?

Reply:
Seems like it's used here
https://github.com/sneezymud/sneezymud/blob/master/code/code/misc/discipline.cc#L2928-L2949
Not sure when and how that's used. it exists in all the disc definitions

Reply:
Doesn't look necessary in the slightest. Let's get it merged anyway.